honeybee_schema/energy/simulation.py

```diff
@@ -317,7 +341,7 @@ class SizingParameter(NoExtraBaseModel):
         'the design days on this sizing parameter object.'
     )
 
-    building_type: str = Field(
+    building_type: BuildingTypes = Field(
```

Comment:
Ah crud. I should have caught this. I'm going to have to revert this line as it will break several workflows. This one needs to be a string.